docker-vertica/Dockerfile

```diff
@@ -68,15 +70,15 @@ RUN set -x \
 # versions at once.
```

Comment:
We should update this comment to explain what we are doing with NO_SSH_KEYS. 
```suggestion
# versions at once. This step is required if not including SSH keys because
# conditional copy isn't a thing in Docker. If no SSH keys are to be included, 
# then the next RUN will remove the key we just added.
```